src/pages/PostPage.tsx

```diff
@@ -1,131 +1,191 @@
-            좋아요 3개
-          </div>
-        </div>
-
-        <div className="relative w-full bg-gray-100 p-[1.5rem]">
-          <div className="h-8 font-OAGothic text-sm font-medium text-black">
-            댓글 2개
-          </div>
-          <div className="my-4 flex flex-col">
             <div className="flex">
-              <div className="m-1 h-[3.125rem] w-[3.125rem] rounded-full bg-gray-200" />
-              <div className="m-2">
-                <div className="flex">
-                  <div className="font-OAGothic text-sm font-medium">
-                    혼콕마스터
-                  </div>
-                  <div className="my-auto p-1 font-OAGothic text-[0.625rem] font-medium text-gray-400">
-                    1분 전
-                  </div>
+              <Avatar size="small" />
+              <div className="m-1 flex h-[2.25rem] flex-col justify-center ">
+                <div className="font-OAGothic text-sm font-medium text-gray-500">
+                  {post.author.fullName}
                 </div>
-                <div className="font-OAGothic text-[0.813rem] font-medium text-gray-500">
-                  퍼 가요~~
+                <div className="font-OAGothic text-[0.625rem] font-medium text-gray-400">
+                  {timePassed}
                 </div>
               </div>
-              <div className="ml-auto">✦</div>
+              <DotsIcon className="my-auto ml-auto" />
+              {user.posts.find((post: Post) => post._id === postId) ? (
+                <button onClick={handleGoToEditPage}>수정</button>
+              ) : null}
             </div>
-            <div className="flex">
-              <div className="m-1 h-[3.125rem] w-[3.125rem] rounded-full bg-gray-200" />
-              <div className="m-2">
-                <div className="flex">
-                  <div className="font-OAGothic text-sm font-medium">
-                    혼콕마스터
-                  </div>
-                  <div className="my-auto p-1 font-OAGothic text-[0.625rem] font-medium text-gray-400">
-                    1분 전
+          </div>
+          <div className="relative">
+            <Image
+              className="mx-auto h-[15rem] w-[24rem] p-2"
+              src={post.image}
+            />
+            <div className="w-[342px] p-3">{post.content}</div>
+          </div>
+          <Button
+            onClick={handleLike}
+            size="lg"
+            className="m-10 mx-auto flex w-[7rem] justify-center space-x-1 rounded-[6rem] p-2"
+          >
+            <HeartIcon
+              className={`h-[1rem] w-[1.25rem] stroke-sub-red ${
+                post.likes.find((like) => like.user === user._id)
+                  ? 'fill-sub-red'
+                  : 'fill-white'
+              }`}
+            />
+            <div className="text-xs font-medium">좋아요</div>
+            <div className="text-xs font-medium">{post.likes.length}개</div>
+          </Button>
```

Comment:
얘네도 한 번 더 div로 감싸서 표현해도 좋겠다 싶었어요! 그리고 텍스트에도 div 태그를 사용하셨는데, 텍스트의 경우 저는 한 줄을 다 차지하면 p 태그로 감싸고, 인라인 요소면 span 태그를 씁니당!

```suggestion
          <div className="py-10">
            <Button
              onClick={handleLike}
              size="lg"
              className="mx-auto flex justify-center gap-1 rounded-full border-[1px]"
            >
              <HeartIcon
                className={`h-4 w-5 stroke-sub-red ${
                  post.likes.find((like) => like.user === user._id)
                    ? 'fill-sub-red'
                    : 'fill-white'
                }`}
              />
              <span className="text-xs">좋아요 {post.likes.length}개</span>
            </Button>
          </div>
```

이런 식으로 수정하면 더 깔끔할 것 같아요!